wasi/wasi.go

```diff
@@ -18,6 +20,7 @@ const (
 )
 
 type WASIEnvironment struct {
+	args  wasiStringArray
```

Comment:
Use pointer then we don't need to copy `wasiStringArray` in option 
```suggestion
	args *wasiStringArray
```

Reply:
Improved it. Thanks!